docs/examples.rst

```diff
@@ -7,24 +6,22 @@ Installation
 
 This is only required on your first use.
 
-- Ensure you have the following requirements: Pip, Pipenv, and a local copy of pytac.
+- Ensure you have Pip and a local copy of pytac.
```

Comment:
You *don't* need a local copy of Pytac this time - you're downloading it from PyPI